.github/workflows/CLAUDE.md

```
@@ -0,0 +1,258 @@
+# CI Workflow Patterns
+
+## Shell Scripts in GitHub Actions
+
+### Error Handling
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)